armi/reactor/converters/geometryConverters.py

```diff
@@ -1310,12 +1316,14 @@ def convert(self, r=None):
             )
             return r
         elif not (
-            r.core.symmetry == self.EXPECTED_INPUT_SYMMETRY
+            str(r.core.symmetry) == self.EXPECTED_INPUT_SYMMETRY
```

Comment:
I changed it to compare as tuples because we can't compare the SymmetryType instances directly. Should we implement a `__eq__` to allow a direct comparison? I don't think it would happen anywhere else other than here, so it might not be worthwhile.